docs/03-00-installation.md

```diff
@@ -129,9 +126,9 @@ Then execute the following commands:
 
 ```
 bundle install
+nvm install
```

Comment:
This requires you to have nvm previously installed, we might not want to enforce this.